app/models/throttle.rb

```diff
@@ -13,28 +13,28 @@ class Throttle < ApplicationRecord
 
   THROTTLE_CONFIG = {
     idv_acuant: {
-      max_attempts: (AppConfig.env.acuant_max_attempts || 3).to_i,
-      attempt_window: (AppConfig.env.acuant_attempt_window_in_minutes || 360).to_i,
+      max_attempts: AppConfig.env.acuant_max_attempts.to_i,
+      attempt_window: AppConfig.env.acuant_attempt_window_in_minutes.to_i,
     },
     reg_unconfirmed_email: {
-      max_attempts: (AppConfig.env.reg_unconfirmed_email_max_attempts || 20).to_i,
-      attempt_window: (AppConfig.env.reg_unconfirmed_email_window_in_minutes || 60).to_i,
+      max_attempts: AppConfig.env.reg_unconfirmed_email_max_attempts.to_i,
+      attempt_window: AppConfig.env.reg_unconfirmed_email_window_in_minutes.to_i,
     },
     reg_confirmed_email: {
-      max_attempts: (AppConfig.env.reg_confirmed_email_max_attempts || 20).to_i,
-      attempt_window: (AppConfig.env.reg_confirmed_email_window_in_minutes || 60).to_i,
+      max_attempts: AppConfig.env.reg_confirmed_email_max_attempts.to_i,
+      attempt_window: AppConfig.env.reg_confirmed_email_window_in_minutes.to_i,
     },
     reset_password_email: {
-      max_attempts: (AppConfig.env.reset_password_email_max_attempts || 20).to_i,
-      attempt_window: (AppConfig.env.reset_password_email_window_in_minutes || 60).to_i,
+      max_attempts: AppConfig.env.reset_password_email_max_attempts.to_i,
+      attempt_window: AppConfig.env.reset_password_email_window_in_minutes.to_i,
     },
     idv_resolution: {
-      max_attempts: (AppConfig.env.idv_max_attempts || 5).to_i,
-      attempt_window: (AppConfig.env.idv_attempt_window_in_hours || 6).to_i * 60,
+      max_attempts: AppConfig.env.idv_max_attempts.to_i,
+      attempt_window: AppConfig.env.idv_attempt_window_in_hours.to_i * 60,
     },
     idv_send_link: {
-      max_attempts: (AppConfig.env.idv_send_link_max_attempts || 5).to_i,
```

Comment:
are all of these set in the config? making them required will also apply to lower environments even if they're not used.

Reply:
They're all in the defaults. I double checked that.